test/uk/gov/hmrc/softdrinksindustrylevy/connectors/DesConnectorSpec.scala

```diff
@@ -132,52 +146,63 @@ class DesConnectorSpecBehavioural extends WiremockSpec {
         post(urlEqualTo("/soft-drinks/subscription/utr/11111111119"))
           .willReturn(aResponse().withStatus(429)))
 
-      lazy val response = the[Exception] thrownBy (TestDesConnector
+      lazy val response = the[Exception] thrownBy (desConnector
         .createSubscription(sub, "utr", "11111111119")
         .futureValue)
       response.getMessage must startWith(
         "The future returned an exception of type: uk.gov.hmrc.http.Upstream5xxResponse")
-    }
+    }*/
 
     "displayDirectDebit should return Future true when des returns directDebitMandateResponse set to true" in {
-      stubFor(
-        get(urlEqualTo("/cross-regime/direct-debits/zsdl/zsdl/XMSDIL000000001"))
-          .willReturn(aResponse().withBody("""{ "directDebitMandateFound" : true }""").withStatus(200)))
-      val response = TestDesConnector.displayDirectDebit("XMSDIL000000001").futureValue
-      response.directDebitMandateFound mustBe true
+      when(mockHttpClient.GET[HttpResponse](any(), any(), any())(any(), any(), any()))
+        .thenReturn(Future.successful(HttpResponse(200, """{ "directDebitMandateFound" : true }""")))
+
+      val response = desConnector.displayDirectDebit("XMSDIL000000001")
+      response.map { directDebitMandateFound =>
```

Comment:
using `map` is not a good idea here as the existing one `.futureValue` is correct